WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/EditableOrderViewModelTests.swift

```diff
@@ -2,6 +2,7 @@ import XCTest
 @testable import WooCommerce
 import Yosemite
 import WooFoundation
+import Networking
```

Comment:
Since we are actually importing that `Networking` `DotComError`in `EditableOrderViewModel`, I think its fine if we replicate that behavior and import `Networking` in `WooCommerce`. I changed the import a little bit to `import enum Networking.DotcomError` to make it more specific as in the `EditableOrderViewModel`.